BetterHolidays/markets/sse.py

```
@@ -0,0 +1,35 @@
+from .market import Market
+import datetime as dt
+from ..days import Holiday
```

Ruff (0.8.2)

`..days.Holiday` imported but unused

Remove unused import: `..days.Holiday`

(F401)